Minimal/Calculus.lean

```diff
@@ -434,6 +482,116 @@ namespace PReduce
                 | void => Premise.consVoid b premise
                 | attached u' => Premise.consAttached b u' u' (prefl u') premise
               )
+
+  def singlePremiseInsert
```

Comment:
Weird indentation again?

Reply:
This is due to PReduce namespace. Premise is defined there, it is mutual to PReduce's definition, so all properties of Premise are also in the namespace